lib/Dialect/Calyx/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+##===- CMakeLists.txt - Calyx dialect code --------------------*- cmake -*-===//
+##
+## Implementation files the Calyx dialect.
+##
+##===----------------------------------------------------------------------===//
+
+set(srcs
+        CalyxDialect.cpp
```

Comment:
nit: is this spacing intented? seems a little over-indented, but it could just be me.